examples/bri-3/src/bri/merkleTree/agents/import MerkleTree from 'merkletreejs';.ts

```diff
@@ -0,0 +1,20 @@
+import MerkleTree from 'merkletreejs';
```

Comment:
@ognjenkurtic Does this file serve a purpose or was it for dev testing?

Reply:
Ah sorry, will remove it

Reply:
Removed